src/components/index.ts

```diff
@@ -4,6 +4,7 @@ export * from './Button';
 export * from './Card';
 export * from './Collapsible';
 export * from './Datatable';
```

Comment:
@ajkl2533 Do you plan to remove old Datatable from codebase?

Reply:
sure I do, after migration